jobs/job-base-images/python/stalker_job_sdk/stalker_job_sdk/__init__.py

```diff
@@ -1,12 +1,17 @@
+import httpx
 import json
 import sys
 from abc import ABC
 from ipaddress import ip_address
 from os import getenv
+from functools import lru_cache
 
-import httpx
 
 
+@lru_cache
+def get_http_client():
+    return httpx.Client(verify=False, http2=True)
```

Comment:
This should prevent issues when calling the orchestrator many times

Reply:
C'est du pur genie :100: ca peut definitivement avoir des impacts positifs importants sur la performance